File: player.py
class Player:
    def __init__(self, player_name: str = "Player"):
        self.display = player_name
        self.location = None
        self.inventory = {}
        self.equipped_items = {}
        self.money = 0

    def has_item(self, item: str, num: int):
        if item in self.inventory:
            if self.inventory[item] >= num:
                return True

        return False

    def add_item(self, item: str, num: int):
        if item in self.inventory:
            self.inventory[item] += num
        else:
            self.inventory[item] = num

    def reset_equips(self):
        self.equipped_items = {
            "head": "",
            "neck": "",
            "torso": "",
            "left_hand": "",
            "right_hand": "",
            "belt": "",
            "legs": "",
            "feet": "",
        }

File: app/static/js/data/player.js
class Player{
    constructor(player_name = "Player"){
        this.display = player_name
        this.location = null
        this.inventory = {}
        this.equipped_items = {}
        this.money = 0
	}	

    has_item(item, num){
        if (item in this.inventory){
			if (this.inventory[item] >= num){
				return true
			}
		}
        return false
	}

    add_item(item, qty){
        if (item in this.inventory){
            this.inventory[item] += qty
		} else{
            this.inventory[item] = qty
		}
	}

    reset_equips(){
        this.equipped_items = {
            head: null,
            neck: null,
            torso: null,
            left_hand: null,
            right_hand: null,
            belt: null,
            legs: null,
            feet: null,
        }
	}	
}